package.json

```diff
@@ -184,7 +187,9 @@
       "**/@mui/monorepo"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "autoprefixer": "^10.4.16"
```

Comment:
Nitpick: Should be in devDependencies?

Reply:
Yesterday I ran into a missing `autoprefixer` dependency on the docs build in Toolpad. It went away after I recreated the lockfile. If this is the result of a similar issue, you may want to try to recreate the lockfile as well.